concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1370,34 +1370,25 @@ jobs:
                 < cf-tfstate/cf.tfstate \
                 > cf-terraform-outputs.yml
 
-                if [ -s cf-vars-store/cf-vars-store.yml ]
-                then
-                  echo cf-vars-store.yml has vars to copy.
-                else
-                  echo cf-vars-store.yml is empty. Skipping ...
-                  exit 0
-                fi
-
+                echo 'Logging into Credhub'
+                credhub api -s "$BOSH_ENVIRONMENT:8844/api"
+                credhub login --client-name="${CREDHUB_CLIENT_ID}" --client-secret="${CREDHUB_CLIENT_SECRET}"
 
                 echo 'Getting variables'
                 CF_ADMIN=admin
-                CF_PASS=$($VAL_FROM_YAML cf_admin_password cf-vars-store/cf-vars-store.yml)
-                CF_CLIENT_SECRET=$($VAL_FROM_YAML secrets_uaa_clients_paas_billing_secret cf-vars-store/cf-vars-store.yml)
+                CF_PASS=$(credhub get -q -n /${DEPLOY_ENV}/${DEPLOY_ENV}/cf_admin_password)
```

Comment:
For the sake of reducing the amount of visual clutter, could we swap the duplicating of `DEPLOY_ENV` for

```
CREDHUB_NS="/${DEPLOY_ENV}/${DEPLOY_ENV}"
CF_PASS=$(credhub get -qn ${CREDHUB_NS}/cf_admin_password)
```